File: app/src/main/java/com/fastaccess/data/service/TrendingService.java
package com.fastaccess.data.service;


import android.support.annotation.Nullable;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Path;
import retrofit2.http.Query;

/**
 * Created by Kosh on 02 Jun 2017, 12:58 PM
 */

public interface TrendingService {

    @GET("{lan}") Observable<String> getTrending(@Path("lan") @Nullable String lan, @Nullable @Query("since") String since);
}

File: app/src/main/java/com/fastaccess/data/service/trending/TrendingService.kt
package com.fastaccess.data.service.trending

import com.fastaccess.data.dao.kot.TrendingResponse
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Query

/**
 * Created by Kosh on 30 May 2017, 11:20 PM
 */

interface TrendingService {
    @GET("v2/trending") fun getTrending(@Query("lan") query: String, @Query("since") since: String): Observable<List<TrendingResponse>>
}